src/app/basic-form/basic-form-success.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import { FormBuilder, Validators } from '@angular/forms';
```

Comment:
You can completely remove lines 2 and 3 here

Reply:
Done